helloworld-impl/src/main/java/sample/helloworld/impl/HelloServiceImpl.java

```diff
@@ -37,32 +33,22 @@
   }
 
   @Override public ServiceCall<NotUsed, String> hello(String id) {
-    System.out.println("hello");
     return request -> {
-      // Look up the hello world entity for the given ID.
       PersistentEntityRef<HelloCommand> ref = persistentEntityRegistry.refFor(HelloWorld.class, id);
-      // Ask the entity the Hello command.
-      greetingsTopic();
       return ref.ask(new Hello(id, Optional.empty()));
     };
   }
 
   @Override public ServiceCall<GreetingMessage, Done> useGreeting(String id) {
-    System.out.println("usegreeting");
     return request -> {
-      // Look up the hello world entity for the given ID.
       PersistentEntityRef<HelloCommand> ref = persistentEntityRegistry.refFor(HelloWorld.class, id);
-      // Tell the entity to use the greeting message specified.
       return ref.ask(new UseGreetingMessage(request.message));
     };
 
   }
 
   @Override
-  //#implement-topic
   public Topic<GreetingMessage> greetingsTopic() {
-    System.out.println("greetingsTopic");
-    //throw new UnsupportedOperationException("Missing implementation");
     return TopicProducer.singleStreamWithOffset(offset -> persistentEntityRegistry.eventStream(HelloEventTag.INSTANCE, offset)
         .map(this::convertEvent));
   }
```

Comment:
@ignasi35 Thanks for pointing it out. I was making a very trivial mistake at the time of making curl. 
Thanks again for helping me and correcting my mistakes. :slightly_smiling_face: